DeAceroDLabs/marketplace-frontend
# fix: [DMP-688] Deleted ayuda and fixed up footer styles.

## What is done

[[DMP-688] Deleted ayuda and fixed up footer styles](https://deacero.atlassian.net/browse/DMP-688)

## How is done

- Added ayuda button from `Footer` component
- Fixed up styles so footer and Hola, your name buttons are the same as the design.

## How to test

1. Run the project via `npm start`.
2. You should see the footer correctly.
3. Click on Hola, yourname, these buttons should also look the same as the design.

## Evidence

Before:

![image](https://user-images.githubusercontent.com/81311646/200438578-f4957d68-8150-48ce-8084-c089d8e3bcb5.png)

![image](https://user-images.githubusercontent.com/81311646/200438609-505f0959-3089-4261-a49f-585127f37f2d.png)

After:
![image](https://user-images.githubusercontent.com/81311646/200438665-c4db5672-2616-45e2-a024-7d8f4c096cb6.png)

![image](https://user-images.githubusercontent.com/81311646/200438642-ffb3b97d-2e28-4e6e-a121-8578624382a5.png)
